demo/twenty_news/build.gradle

```diff
@@ -17,15 +17,10 @@ dependencies {
   compile 'org.slf4j:slf4j-simple:1.7.7'
 }
 
-compileScala {
```

Comment:
Don't think anyone here has more experience with gradle than you do ;) I don't have a strong opinion either.